src/main/java/com/petpool/application/util/ip/IpParserImpl.java

```diff
@@ -7,22 +7,27 @@
 import java.util.List;
 
 public class IpParserImpl implements IpParser {
+    private static String DEFAULT_PARSE_VALUE = "";
+
     @Override
     public @NotNull String parse(HttpHeaders headers) {
         if (headers == null) {
-            return "";
+            return DEFAULT_PARSE_VALUE;
         }
+
         InetSocketAddress address = headers.getHost();
+
         if (address != null) {
             String ip = address.getHostName();
-            if (ip != null && !(ip.equals(""))) {
+            if (ip != null && !(ip.isEmpty())) {
```

Comment:
`StringUtils.isEmpty` is handy instead of explicit checks for both null and emptiness. [API documentation Apache Commons StringUtils::isEmpty](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isEmpty-java.lang.CharSequence-)